Validation/RecoTrack/src/SiStripTrackingRecHitsValid.cc

```diff
@@ -815,6 +729,7 @@ void SiStripTrackingRecHitsValid::analyze(const edm::Event & e, const edm::Event
 	    if( (min(rechitpro.clusiz, 4) - 1) == 3 ){fillME(iLayerME->second.meResolxMFRphiwclus3,sqrt(rechitpro.resolxxMF));}
 	    if( (min(rechitpro.clusiz, 4) - 1) == 4 ){fillME(iLayerME->second.meResolxMFRphiwclus4,sqrt(rechitpro.resolxxMF));}
```

Comment:
This is part of the preexisting code.
Here is a logic error and I will correct in the next pull request for this class.

The problem is that the max from <code> min(rechitpro.clusiz, 4) </code>  is 4, therefore the max of the query is

    if ( (4 - 1) == 4 )

so this <code> fillME </code> will never be executed


________________________________________________________________________

Page 13 from this slides list possible solutions:
https://indico.cern.ch/event/394132/session/1/contribution/5/attachments/1140008/1632665/code_cleanup.pdf